dags/openshift_nightlies/config/benchmarks/osp-large-control-plane.json

```diff
@@ -32,13 +32,20 @@
         },
         {
             "name": "node-density",
-            "workload": "kube-burner",
-            "custom_cmd": "kube-burner ocp node-density --uuid=${UUID} --pods-per-node=245 --timeout=2h --pod-ready-threshold=10s --es-server=${ES_SERVER} --es-index=ripsaw-kube-burner --user-metadata=metadata.yml"
+            "workload": "kube-burner-ocp-wrapper",
+            "env": {
+                "WORKLOAD": "node-density",
+                "EXTRA_FLAGS": "--pod-ready-threshold=10s"
+            }
         },
         {
-            "name": "cluster-density",
-            "workload": "kube-burner",
-            "custom_cmd": "kube-burner ocp cluster-density --uuid=${UUID} --iterations=3000 --timeout=6h --es-server=${ES_SERVER} --es-index=ripsaw-kube-burner --user-metadata=metadata.yml"
+            "name": "cluster-density-v2",
+            "workload": "kube-burner-ocp-wrapper",
+            "env": {
+                "WORKLOAD": "cluster-density-v2",
+                "ITERATIONS": "216",
```

Comment:
@rsevilla87 shouldnt this number be bigger, i.e. 2268? 216 seems really small for the large test

Reply:
2016 iterations is 8 its/node and gave a good result. I support that number from a possible omitted '0'.

Reply:
i was basing off of 9xnodes which seemed to be what the other iteration values were mathed out to be. I'm fine w/ either I just assume the 216 was a bad copy/paste :-)

Reply:
yeah, it's a mistake :), fixing it